ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.21__multi-nightly.yaml

```diff
@@ -2172,6 +2172,34 @@ tests:
     - chain: cert-manager-custom-api-ingress-cert
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-disk-encryption
+- as: gcp-ipi-oidc-mac-no-actas-mini-perm-amd-f28-destructive
```

Comment:
Does `mac` means `machine`? if that, how about `gcp-ipi-oidc-machine-no-actas-mini-perm-amd-f28-destructive` or `gcp-ipi-oidc-no-actas-mini-perm-amd-f28-destructive` if name length exceeds the limit?